src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -338,6 +350,14 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
                     survivalPlotLeftTruncationChecked: this.props
                         .survivalPlotLeftTruncationChecked,
                 };
+                break;
+            }
+            case ChartTypeEnum.MUTATION_DIAGRAM: {
+                controls = {
+                    showResultsPageButton: true,
+                    showXAxisAtTop: true,
+                };
+                break;
```

Comment:
ideally we shouldn't keep adding more cases into the ChartContainer. It should be a generic container with customizable options, so that we can pass mutation specific options as props. This component is getting as big as the study view store itself. At this point it is not very easy to refactor it though. We should probably tackle this issue later.